pkgs/top-level/all-packages.nix

```diff
@@ -33282,6 +33282,8 @@ with pkgs;
 
   ### MISC
 
+  argononed = callPackage ../misc/drivers/argononed { };
```

Comment:
```suggestion
  argononed = callPackage ../misc/drivers/argononed { };
```
This is out of alphabetical order.

Reply:
@Misterio77 Can you fix this? Otherwise LGTM.